cmd/ova-serial-api/main.go

```diff
@@ -1,56 +1,52 @@
 package main
 
 import (
-	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
 	"google.golang.org/grpc"
 	"net"
-	"os"
 	server "ova-serial-api/internal/api"
 	"ova-serial-api/internal/config"
 	api "ova-serial-api/pkg/ova-serial-api"
 	"time"
 )
 
 const (
-	configPath         = "config/test_config.json"
+	configPath         = "config/test_config.yaml"
 	confUpdIntervalSec = 10
 	grpcPort           = ":82"
 )
 
 func main() {
-	output := zerolog.ConsoleWriter{Out: os.Stdout}
-	log := zerolog.New(output).With().Timestamp().Logger()
-
 	var cfg config.Config
 	go func() {
 		for {
 			err := config.UpdateConfig(configPath, &cfg)
 			if err != nil {
-				log.Error().Msgf("Error while reading config: %s\n", nil)
+				log.Error().Msgf("Error while reading config: %s\n", err)
 			} else {
-				log.Debug().Msgf("Config '%s' updated: %+v\n", configPath, cfg)
+				log.Debug().Msgf("Config '%s' updated: %+v\n", configPath, cfg.Data)
 			}
 
 			time.Sleep(confUpdIntervalSec * time.Second)
 		}
 	}()
 
-	if err := run(&log); err != nil {
+	if err := run(); err != nil {
 		log.Fatal().Msgf("Error while starting server: %v", err)
 	}
 }
 
-func run(logger *zerolog.Logger) error {
+func run() error {
```

Comment:
Давай более понятное название дадим, например что-нибудь такое
```suggestion
func startGRPCServer() error {
```